packages/website/tests/index.spec.ts

```diff
@@ -2,6 +2,16 @@ import AxeBuilder from '@axe-core/playwright';
 import { expect, test } from '@playwright/test';
 
 test.describe('Website', () => {
+  test.beforeEach(async ({ context }) => {
+    // Sponsor logos are sometimes changed or removed between deploys
+    await context.route('https://images.opencollective.com/**/*.png', route =>
+      route.fulfill({
+        status: 200,
+        body: '',
+      }),
+    );
+  });
```

Comment:
Would you be up for splitting this out too? I filed a separate issue at #7332. That way we can get this fixed sooner.